### DISCUSSION TITLE
Suggestion: Separate "Save" and "Close" behavior

### DISCUSSION THREAD

**Discussion by IwateKyle** [pocketbase/pocketbase#2302] [Q&A] [answered]
I wasn't sure where to place this.  It's not really a question _unless_ there is another way to do the same thing.

Current use case: 
I use PocketBase's Rich Text Editor feature for adding longer text.  Imagine a blog post.  I want to be able to save my entry periodically while continuing to edit.  

It would be nice if the "save" feature for the record would keep the record editing window open with a separate close button to allow uninterrupted editing.

**Comment by ganigeorgiev** [comment, marked as answer]
Hm, I think adding another button may clutter the interface, but I've added it to the [roadmap](https://github.com/orgs/pocketbase/projects/2/views/1?pane=issue&itemId=25752846) to consider it (_eventually it could be implemented as a dropdown option and/or combined with "Ctrl+S" shortcut_)

**Reply by IwateKyle** [reply]
I totally understand taking the overall user experience into consideration.  Thanks for adding this to your list of things to think about.
Yes, the shortcut Ctrl-S would be great.  My suggestion of a button is just an illustration of the use case.  I also thought an extra button might look cluttered.
